packages/SwingSet/src/kernel/kernel.js

```diff
@@ -678,7 +681,21 @@ export default function buildKernel(kernelEndowments) {
     return vatKeeper.vatStats();
   }
 
-  async function start(bootstrapVatName, argvString) {
+  const knownBundles = new Map();
```

Comment:
eventually these bundles will want to move into kernelDB, or maybe a separate store of some sort, but definitely on-disk rather than in-memory

Reply:
Yes; this is a transitional solution.